evaluator/images/java/entry.py

```diff
@@ -0,0 +1,179 @@
+#!/usr/local/bin/python3
+import dataclasses
+import os
+import subprocess
+import json
+import re
+import glob
+from collections import defaultdict
+from pathlib import Path
+from typing import Dict, List, Optional
+import xml.etree.ElementTree as ET
+
+import bleach
+
+
+@dataclasses.dataclass
+class BuildResult:
+    success: bool
+    output: str
+    comments: Optional[Dict[str, List[str]]] = dataclasses.field(default_factory=dict)
+    tests: Optional[List[dict]] = dataclasses.field(default_factory=list)
+
+    @staticmethod
+    def fail(error: str) -> "BuildResult":
+        return BuildResult(
+            success=False,
+            output=error,
+        )
+
+
+def parse_tests_report(path):
+    try:
+        for path in glob.glob(os.path.join(path, "*.xml")):
+            tree = ET.parse(path)
+            os.unlink(path)
+
+            tests = []
```

Comment:
This will return only tests from the last `xml` file. Either `tests = []` shold be defined before the cycle (if tests from all `.xml` files should be returned), or `return tests` should be indented one level to the right (if only the tests from the first `.xml` file should be returned).